### ISSUE TITLE
Frozen Friends List and Miiverse

### ISSUE THREAD

**Issue by peter1357908** [Plailect/Guide#134]
Thanks for the awesome guide! Now I have a fresh CFW Sysnand on 11.0.0-32U! Nevertheless, when I attempt to open the Friends List and Miiverse app, my n3DSxl just freezes on the lower-screen display of the colored strip of the icon and name of the app. However, Eshop, Mii Maker and other system titles seemed to work unimpaired...

I saw an earlier issue concerning forgetting to unlink NNID before the entire process, and actually I didn't unlink NNID before the whole thing either...

So was this problem caused by my carelessness to not unlink NNID first?
Is there any way to fix this issue and how can I do it?

Thanks again!

**Comment by Plailect** [comment]
This is a bug fixed by using the [latest Luma hourly builds](http://astronautlevel2.github.io/Luma3DS/)

**Comment by peter1357908** [comment]
I am actually on the latest Luma hourly build.. if the one from 5/14/2016 is the latest one. I completed the guide yesterday and I updated the Luma3DS to the latest hourly build of "afc6f51." Are there any other possible mistakes I made that could cause such problems? I shall also try to re-update my Luma3DS now.
Thanks!

**Comment by OTFree** [comment]
Followed the nintendo restore, and restored my NAND back up, now I was able to update cfw SysNAND to 11.0! I'm having the same issue though, even with updated Luna3DS hourly build, Nintendo Network ID settings keeps having errors and eShop does the same thing.

**Comment by Plailect** [comment]
Ask AuroraWright then, I'm not sure

**Comment by peter1357908** [comment]
I got it "fixed!" It seemed all I needed was patience. I'm not sure if any action of mine actually solved the problem, but after I posted the issue on Luma3DS GitHub page, I tried another possible solution of just "letting Friends List sit there and fix itself" and after 5 minutes-ish of display of the icon strip, the Friends List worked just as fine as before the arm9loaderhax installation. Then Miiverse started to function normally too. It was just amazingly that easy, wheewwwww! :)
